src/modules/assets/assets.service.ts

```diff
@@ -0,0 +1,129 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Provider } from '../../common/provider';
+import { IdentityManager } from '../../ethers/IdentityManager';
+import { IdentityManagerFactory } from '../../ethers/IdentityManagerFactory';
+import { OfferableIdentityFactory } from '../../ethers/OfferableIdentityFactory';
+import { DIDService } from '../did/did.service';
+import { AssetDto } from './assets.dto';
+import { Asset } from './assets.entity';
+
+@Injectable()
+export class AssetsService {
```

Comment:
@dwojno Could have some sync of assets? Is that possible to implement? I guess we need to go through the logs of the `IdentityManager`?

Reply:
Yes of course, currently I'm working on this :)